### ISSUE TITLE
Why 480x640?

### ISSUE THREAD

**Issue by anguoyang** [Howave/RetinaFace-TVM#4]
The image shape of Retina face is :640x480, the aspect ratio is same with normal camera, but in your source code, it is 480x640, is it really correct? thank you

**Comment by Howave** [comment]
you can define your own resolution according to your cases

**Comment by anguoyang** [comment]
yes, I changed it into other resolution, it was fine(which means no errors, the result size is correct), but the scale ratio seems not correct, even use your default 480x640, the bounding box is not correct after multiply by the scale ratio, it is strange that the landmarks are correct after multiplied by the same scale:)
